src/components/reusable_components/InvestItemList.js

```diff
@@ -16,27 +29,183 @@ export default function InvestItemList(props) {
         </View>
       ) : (
         <View style={styles.listItems}>
-          {listData.map((item) => {
-            return (
-              <Text key={item.id} style={styles.text}>
-                {item.name}
+          {listData.length === 0 ? (
+            <View style={styles.noResultsContainer}>
+              <Text style={styles.text}>
+                You don't have any investors yet. Press the plus icon above to
+                create one.
               </Text>
-            );
-          })}
+            </View>
+          ) : (
+            <ScrollView>
+              <Animated.View entering={SlideInDown}>
+                {listData.map((item) => {
+                  return (
+                    <TouchableWithoutFeedback
+                      key={item.id}
+                      onPress={() => console.log("Opening: ", item.name)}
+                    >
+                      <View style={styles.listItem}>
+                        <View style={styles.nameContainer}>
+                          <Text style={styles.listItemName}>{item.name}</Text>
+                        </View>
+                        <View style={styles.imageContainer}>
+                          <Image
+                            style={styles.investorImage}
+                            source={investorImagePathList[item.imageId]}
+                          />
+                        </View>
+                        <View style={styles.indicatorAndStockContainer}>
+                          <View style={styles.indicatorCol}>
+                            <Text style={styles.indictorAndStockHeaderText}>
+                              Indicators
+                            </Text>
+                            <ScrollView>
+                              <View style={styles.indicatorAndStockItems}>
+                                {item.indicators.map((item) => {
+                                  return (
+                                    <Text
+                                      key={item}
+                                      style={styles.indictorAndStockText}
+                                    >
+                                      {item}
+                                    </Text>
+                                  );
+                                })}
+                              </View>
+                            </ScrollView>
+                          </View>
+                          <View style={styles.stockCol}>
+                            <Text style={styles.indictorAndStockHeaderText}>
+                              Stocks
+                            </Text>
+                            <ScrollView>
+                              <View style={styles.indicatorAndStockItems}>
+                                {item.stocks.map((item) => {
```

Comment:
Talked to Jason about this , what happens if an investor has like 10 stocks? There would need to be like a `+x more` at the bottom of the list.

Reply:
Its wrapped in a ScrollView so if the investor has 20 stocks for example the user can scroll up and down within the stock section of the investor card to see all the stocks.